script/build_release.sh

```diff
@@ -4,4 +4,5 @@ if ! [ -z "$1" ]
   then
     mkdir dist/$1
     cp dist/main.min.css dist/$1/main.min.css
+    cp assets/ dist/$1/
```

Comment:
I am not sure this is going to work as intended. For one thing, if you want to copy the directory you'll need to use the `-r` flag for recursion. I'm also not sure this will drop things in the right place (which I'm assuming is something like /dist/0.1/assets/

Reply:
So I was thinking the s3 directory structure should be:
- main.min.css
- v0.1/
- v0.1/main.min.css
- v0.1/fonts/
- v0.1/img/

But it occurs to me that the fonts and images should probably mirror the approach of being both in the root AND in the versioned folder.

Reply:
so I think you'll need two additional lines here:
```
cp -r assets/ dist/
cp -r assets/ dist/$1/
```